src/lib/user-features.ts

```diff
@@ -39,174 +21,17 @@ const SUPPORT_RECURRING_CONTRIBUTOR_ANNUAL_COOKIE =
-	if (JsonResponse.alertAvailableFor) {
-		setCookie({
-			name: ACTION_REQUIRED_FOR_COOKIE,
-			value: JsonResponse.alertAvailableFor,
-		});
-	}
-
-	if (JsonResponse.contentAccess.digitalPack) {
-		setAdFreeCookie(2);
-	} else if (adFreeDataIsPresent() && !forcedAdFreeMode) {
-		removeCookie({ name: AD_FREE_USER_COOKIE });
-	}
-};
-
-const deleteOldData = (): void => {
-	// We expect adfree cookies to be cleaned up by the logout process, but what if the user's login simply times out?
-	removeCookie({ name: AD_FREE_USER_COOKIE });
-	removeCookie({ name: USER_FEATURES_EXPIRY_COOKIE });
-	removeCookie({ name: PAYING_MEMBER_COOKIE });
-	removeCookie({ name: RECURRING_CONTRIBUTOR_COOKIE });
-	removeCookie({ name: ACTION_REQUIRED_FOR_COOKIE });
-	removeCookie({ name: DIGITAL_SUBSCRIBER_COOKIE });
-	removeCookie({ name: HIDE_SUPPORT_MESSAGING_COOKIE });
-	removeCookie({ name: ONE_OFF_CONTRIBUTION_DATE_COOKIE });
-};
-
-const requestNewData = () => {
-	return getAuthStatus()
-		.then((authStatus) =>
-			authStatus.kind === 'SignedInWithCookies' ||
-			authStatus.kind === 'SignedInWithOkta'
-				? authStatus
-				: Promise.reject('The user is not signed in'),
-		)
-		.then((signedInAuthStatus) => {
-			return fetchJson(
-				`${
-					window.guardian.config.page.userAttributesApiUrl ??
-					'/USER_ATTRIBUTE_API_NOT_FOUND'
-				}/me`,
-				{
-					mode: 'cors',
-					...getOptionsHeadersWithOkta(signedInAuthStatus),
-				},
-			)
-				.then((response) => {
-					if (!validateResponse(response)) {
-						throw new Error('invalid response');
-					}
-					return response;
-				})
-				.then(persistResponse)
-				.catch(noop);
-		});
-};
-
-const featuresDataIsOld = () =>
-	cookieIsExpiredOrMissing(USER_FEATURES_EXPIRY_COOKIE);
-
 const isDigitalSubscriber = (): boolean =>
 	getCookie({ name: DIGITAL_SUBSCRIBER_COOKIE }) === 'true';
 
-const userNeedsNewFeatureData = (): boolean =>
-	featuresDataIsOld() || (isDigitalSubscriber() && !adFreeDataIsPresent());
-
-const userHasDataAfterSignout = async (): Promise<boolean> =>
-	!(await isUserLoggedInOktaRefactor()) && userHasData();
-
-/**
- * Updates the user's data in a lazy fashion
- */
-const refresh = async (): Promise<void> => {
-	if ((await isUserLoggedInOktaRefactor()) && userNeedsNewFeatureData()) {
-		return requestNewData();
-	} else if ((await userHasDataAfterSignout()) && !forcedAdFreeMode) {
-		deleteOldData();
-	}
-	return Promise.resolve();
-};
-
 const supportSiteRecurringCookiePresent = () =>
 	getCookie({ name: SUPPORT_RECURRING_CONTRIBUTOR_MONTHLY_COOKIE }) != null ||
 	getCookie({ name: SUPPORT_RECURRING_CONTRIBUTOR_ANNUAL_COOKIE }) != null;
 
-/**
- * Does our _existing_ data say the user is a paying member?
- * This data may be stale; we do not wait for userFeatures.refresh()
- */
+// /**
+//  * Does our _existing_ data say the user is a paying member?
+//  * This data may be stale; we do not wait for userFeatures.refresh()
+//  */
 const isPayingMember = async (): Promise<boolean> =>
```

Comment:
Can IsPayingMember be deleted? I only see it mocked it tests. I don't see anywhere where its functionality is used

Reply:
This is one of the few remaining functions which are needed in testing for `commercial-features.spec.ts` in `isPayingMember.mockResolvedValue(false);`. It can definitely be removed at a later date?